### ISSUE TITLE
Restaurant names with spaces are parsed incorrectly

### ISSUE THREAD

**Issue by soowei** [ChristopherzwTan/slack_memebot#22]
Doing the following command: "lunchbot add restaurant Boston Pizza 3", results in the following response and entry: "Adding restaurant Pizza with weight 3".

**Comment by ChristopherzwTan** [comment]
I think @n-martin has already handled this around line 118?

**Comment by n-martin** [comment]
I don't think this case is handled there actually. The quotations in the post are not part of the command that was typed in.
